code_schemes/new_practises.json

```diff
@@ -0,0 +1,71 @@
+{
+    "SchemeID": "Scheme-048fe72c",
+    "Name": "New practices",
```

Comment:
What is the difference between this and new practices reasons?

Reply:
This has the labels of the new practices that the farmers are willing to implement and the reasons explain the motivation to implement the new practices.

Reply:
new practices and new practices reasons will be two different datasets